news/2023/2023-08-05-skinning-contest-icons-of-an-era-results.md

```diff
@@ -0,0 +1,61 @@
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/dVbEeRi0y_A" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Unfortunately for thetasigma, they were unable to get their [2nd win in a row](/wiki/Contests/Skinning_Contest/3#podium), but by now they have become a household name in the skinning community by repeatedly creating top-of-the-line skins, regardless of the theme. Congratulations on the 3rd place!
+
+![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/3rd_badge@2x.png)
+
+## 2nd Place - Everlasting Moment
+
+Time stood still as **1058** voters cast their votes, resulting in a very close battle between 3rd and 2nd place. In the end, [Chromasia](https://osu.ppy.sh/users/7306251) was able to win the battle and achieve **2nd** place with their entry!
+
+[Everlasting Moment](https://osu.ppy.sh/community/forums/topics/1792308) keeps up with the universe theme, while providing a unique flair based on the background art used in several beatmaps of [Release Hallucination](https://osu.ppy.sh/beatmaps/artists/194)'s *Chronostasis*. The colour palette is in line with the original art, and that makes this skin very pleasing to use!
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/MZHOHYw1ysw" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Considering the fact that this is the 2nd ever skin created by Chromasia for an official skinning contest, we are very impressed with the quality of their work and cannot wait to see what else they have to offer. Congratulations on the 2nd place!
+
+![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/2nd_badge@2x.png)
+
+## Winner - FOOL MOON NIGHT
+
+[Spoo](https://osu.ppy.sh/users/11805037) was not a *fool* when they chose their theme (which somehow again involves the universe or space), because this skin stands head and shoulders above all other entries with a mind-blowing **1303** votes!
+
+Inspired from the [2016 map of *A FOOL MOON NIGHT*](https://osu.ppy.sh/beatmapsets/524026#osu/1186901) by The Koxx, [Spoo's entry](https://osu.ppy.sh/community/forums/topics/1775136?n=1) completely blew the minds of the community by hand-drawing **everything** in the style of the original background. The quality of this skin is impeccable, and the **1st** place is absolutely deserved.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/tYEMQL28rqg" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Spoo is also someone who has only had 2 entries for an official contest, however the effort they put into this skin blew everything else out of the water, as apparent by the 300 vote gap between 1st and 2nd place. Congratulations on your very well deserved first victory!
```

Comment:
> Spoo is also someone who has only had 2 entries for an official contest

This sentences does nothing, should be cut

If you want to do a paragraph about contest entries should probably be smth like "With only their second submission into an official contest they already got the winning spot"

Reply:
Just read 2nd place paragraph (somehow skipped that, oops) and that would be quite similar to it...